CharlieBackend.Root/CompositionRoot.cs

```diff
@@ -63,7 +64,7 @@ public static void InjectDependencies(IServiceCollection services, IConfiguratio
             services.AddScoped<IScheduledEventHandlerFactory, ScheduledEventHandlerFactory>();
             services.AddScoped<IHomeworkStudentService, HomeworkStudentService>();
             services.AddScoped<IEventsService, EventsService>();
-
+            services.AddScoped<IJwtGenerator, JwtGenerator>();
```

Comment:
Why add this service as a scoped and not as a singleton? (Same goes for other services)
There's no point in creating new service instances on each request for the most of these services.

Reply:
It seems not viable because of we might be getting new AuthOptions data each new scope.